nodes/list_struct/start_end.py

```diff
@@ -75,11 +76,11 @@ def count(self, data, level, mode):
         if level:
             for obj in data:
                 out.append(self.count(obj, level-1, mode))
-        elif type(data) in [tuple, list]:
+        elif type(data) in [tuple, list, ndarray]:
```

Comment:
`isinstance(data, (tuple, list, ndarray))`  ?

i don't recall why we moved to using `isinstance`, or if it is better, but it would be tidy to persist.

Reply:
I thought as it did not support inheritance should be faster, now i see is slower :)
I will change it 
BTW your review comment woke up travis but it seems is not as fast as usual :)

Reply:
a bet a lot of people are coding more man-hours at the moment..